MapboxDirections/MBRouteOptions.swift

```diff
@@ -331,8 +331,12 @@ open class RouteOptions: NSObject, NSSecureCoding {
             params.append(URLQueryItem(name: "roundabout_exits", value: String(includesExitRoundaboutManeuver)))
         }
         
-        if !excludedRoadClasses.isEmpty {
-            params.append(URLQueryItem(name: "exclude", value: excludedRoadClasses.description))
+        if !roadClassesToAvoid.isEmpty {
+            let allRoadClasses = roadClassesToAvoid.description.components(separatedBy: ",")
+            if allRoadClasses.count > 1 {
+                print("`roadClassesToAvoid` only accepts one `RoadClasses`. Although \(roadClassesToAvoid.description) was specified, only \(String(describing: allRoadClasses.first)) will be used.")
+            }
+            params.append(URLQueryItem(name: "exclude", value: roadClassesToAvoid.description))
```

Comment:
Given the assertion above, we should include (er, exclude) only the first roadClass.